File: src/main/java/Goorm/Introduce/Domain/FeedBack/FeedBack.java
package Goorm.Introduce.Domain.FeedBack;

import lombok.Data;

/**
 * 페이지 내용이 담길 객체
 */
@Data
public class FeedBack {
    // DB에서 관리할 아이디
    private String id;
    // 이름
    private String name;
    //역활
    private String role;
    // 작성자 프로필 이미지
    private String profile;
    // 날짜
    private String date;
    // 작성 대상: 누구에 대한 피드백인지
    private String memberId;
    // 작성 내용
    private String content;

}

File: src/main/resources/static/assets/js/feedback.js
let feedbackWriteBtn = document.querySelector('.feedbacks .btn-write');
let feedbackModifyBtn = document.querySelectorAll('.feedbacks .btn-modify');
let feedbackDeleteBtn = document.querySelectorAll('.feedbacks .btn-delete');

function feedbackWrite() {
    popupOpenFeedBack('피드백 작성하기', 'board');
}

function feedbackModify(id, role, name, content) {
  popupModifyFeedback('피드백 수정하기', 'board', id, role, name, content);
}

function feedbackDelete(id) {
  const memberId = document.getElementById('memberId').value;
  if(confirm('삭제하시겠습니까?')) {
        let data = {
            id : id,
        };
        httpRequest = new XMLHttpRequest();
        httpRequest.open("DELETE","/board/delete", true);
        httpRequest.responseType = "json";
        httpRequest.setRequestHeader("Content-Type","application/json");
        httpRequest.send(JSON.stringify(data));
         setTimeout(function(){
            window.location.href="/member/"+memberId;
         }, 300);
  }
}
